Feature/sheet view add notes

deonjo opened the pull request:
![Screenshot 2019-07-11 15 05 01](https://user-images.githubusercontent.com/461874/61088477-49aa8780-a3ed-11e9-8b6a-8bdaa57a32b8.png)

Added a column for notes for people in sheet view.

deonjo commented:
This let's you add notes for people. I'm going to extend it, so that they can put information like address, and year/age, gender. It's all the information that you need to arrange people. Notes is just something general.

jeffchiucp commented on src/containers/listView/sheetView.jsx:
```diff
@@ -318,8 +318,6 @@ class SheetView extends Component {
                             rowHeaders={false} 
                             afterChange={this.onCellValueChange}
                             minSpareRows={1}
-                            contextMenu={this.state.contextMenu}
```
why are we removing the dropdown menu?

jeffchiucp commented:
wondering why we remove the ag-Grid & drop-down menu?

jeffchiucp commented:
<img width="1440" alt="Screen Shot 2019-07-11 at 11 24 49 PM" src="https://user-images.githubusercontent.com/10523517/61136249-20830900-a478-11e9-8bf8-7fd8424e512a.png">
<img width="1440" alt="Screen Shot 2019-07-11 at 11 22 32 PM" src="https://user-images.githubusercontent.com/10523517/61136250-20830900-a478-11e9-831f-3f6396440cb4.png">

jeffchiucp commented:
@deonjo wondering why we remove the drop-down menu?

deonjo commented:
We removed AG-Grid, because AG-Grid's community version is limited. You can't do copy and paste. And we have switched completely over to Handsontable instead now.

I removed the dropdown context menu, because i'm not yet ready to support the actions in the dropdown menu.
